### DISCUSSION TITLE
datatables.net version 1.13.6 is serving 1.13.5 files

### DISCUSSION THREAD

**Discussion by DmitriIV100** [cdnjs/cdnjs#14209] [General] [answered]
https://cdnjs.com/libraries/datatables.net/1.13.6

The linked JS files... 

- https://cdnjs.cloudflare.com/ajax/libs/datatables.net/1.13.6/jquery.dataTables.js
- https://cdnjs.cloudflare.com/ajax/libs/datatables.net/1.13.6/jquery.dataTables.min.js

...are the same version as in 1.13.5

![image](https://github.com/cdnjs/cdnjs/assets/105227000/a84b0078-7a6c-4744-9f9c-efa88b1f9298)

The same issue **might** affect the datatables.net-bs5 library.

Thank you

**Comment by MattIPv4** [comment, marked as answer]
cdnjs merely mirrors whatever files were published from the source library. Looking at the files on NPM for 1.13.6, you can see they also contain the 1.13.5 version reference you are seeing here: https://www.npmjs.com/package/datatables.net?activeTab=code

This is an upstream issue with the library itself, not cdnjs.

**Reply by DmitriIV100** [reply]
Thank you

I only checked at the https://datatables.net/ website and the files linked to on the front page are actually 1.13.6

![image](https://github.com/cdnjs/cdnjs/assets/105227000/a8261cb0-4e87-44e7-a428-2e07a4dd61ea)

If/when NPM is updated, will the changes automatically be reflected on CDNJS?

**Reply by MattIPv4** [reply]
No, assets on cdnjs are immutable, they will never change (and I believe the same is true of NPM).
